### PR TITLE
Create Service Handling Units window

### PR DESCRIPTION
**Pull request by BodorAnnamaria**
https://github.com/metasfresh/me03/issues/6257

### CODE REVIEW

File: backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5573610_sys_gh6310_UpdateSysConfigForHUEditorViewFactoryLayout.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+update ad_sysconfig set value = 'N' where name = 'de.metas.ui.web.handlingunits.HUEditorViewFactory.AlwaysUseSameLayout';
```
Comment (metas-ts) [inline backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5573610_sys_gh6310_UpdateSysConfigForHUEditorViewFactoryLayout.sql:1]: i'm not 100% sure what this `AD_SysConfig` does..
it's used in `HUEditorViewFactoryTemplate`, which has five other subclasses, besides `ServiceHUEditorViewFactory`...

:question: ...so, in other words: are we sure this doesn't meddle in bad ways with other HU-Editors?

Also, please consider adding a description to this AD_SysConfig (->boyscout rule)

Comment (BodorAnnamaria) [reply backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5573610_sys_gh6310_UpdateSysConfigForHUEditorViewFactoryLayout.sql]: @metas-ts I removed the script it was not needed

> i'm not 100% sure what this `AD_SysConfig` does..
> it's used in `HUEditorViewFactoryTemplate`, which has five other subclasses, besides `ServiceHUEditorViewFactory`...
> 

 I removed the script it was not needed

**Review by metas-ts — APPROVED** [review]

**Comment by cristinamghita** [comment]
Linked to: https://github.com/metasfresh/metasfresh/issues/10414

File: backend/metasfresh-webui-api/src/main/java/de/metas/ui/web/handlingunits/WEBUI_HU_Constants.java [hunk 2]
```diff
@@ -30,6 +30,9 @@
 	public static final String WEBUI_HU_Window_ID_String = "540189"; // FIXME: hardcoded
 	public static final WindowId WEBUI_HU_Window_ID = WindowId.fromJson(WEBUI_HU_Window_ID_String);
 
+	public static final String WEBUI_SERVICE_HU_Window_ID_String = "541011"; // FIXME: hardcoded
+	public static final WindowId WEBUI_SERVICE_HU_Window_ID = WindowId.fromJson(WEBUI_SERVICE_HU_Window_ID_String);
+
 	public static final WindowId WEBUI_HU_Trace_Window_ID = WindowId.fromJson("540353");
 
 	/**
```
Comment (cristinamghita) [inline backend/metasfresh-webui-api/src/main/java/de/metas/ui/web/handlingunits/WEBUI_HU_Constants.java:38]: Linked to: https://github.com/metasfresh/metasfresh/issues/10414

**Review by cristinamghita — COMMENTED** [review]
Linked to: https://github.com/metasfresh/metasfresh/issues/10414

**Pull request merged**